gpt4all-chat/modellist.cpp

```diff
@@ -1487,16 +1530,101 @@ void ModelList::parseModelsJsonFile(const QByteArray &jsonData, bool save)
             { ModelList::OnlineRole, true },
             { ModelList::DescriptionRole,
              tr("<strong>OpenAI's ChatGPT model GPT-4</strong><br>") + chatGPTDesc + chatGPT4Warn },
-            { ModelList::RequiresVersionRole, "2.4.2" },
+            { ModelList::RequiresVersionRole, "2.7.4" },
             { ModelList::OrderRole, "cb" },
             { ModelList::RamrequiredRole, 0 },
             { ModelList::ParametersRole, "?" },
             { ModelList::QuantRole, "NA" },
             { ModelList::TypeRole, "GPT" },
+            { ModelList::UrlRole, "https://api.openai.com/v1/chat/completions"},
+        };
+        updateData(id, data);
+        
+    }
```

Comment:
Trailing whitespace and unnecessary blank line:
```suggestion
        updateData(id, data);
    }
```